torch/testing/_internal/common_methods_invocations.py

```diff
@@ -18253,6 +18264,26 @@ def __init__(
             }), 'TestCommon', 'test_python_reference_consistency', device_type='cpu'),
         ),
     ),
+    ElementwiseUnaryPythonRefInfo(
+        "_refs.nn.functional.selu",
+        torch_opinfo_name="nn.functional.selu",
+        decorators=(
+            DecorateInfo(toleranceOverride({
```

Comment:
Nice toleranceOverride for CPU -- @ezyang also has a suggestion https://github.com/pytorch/pytorch/issues/77687 that would avoid us having to add these in the future